app/renderer/components/preferences/advancedTab.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../immutableComponent')
+
+const {SettingsList, SettingItem, SettingCheckbox} = require('../settings')
```

Comment:
Can you please reorder imports in something like this: https://github.com/brave/browser-laptop/blob/master/js/components/main.js or https://github.com/brave/browser-laptop/pull/8225/files#diff-169b96dac6a9688438a29c5013735bda

This way it's really easy to find if you are searching for something.